firefoxThemes/popup/index.html

```diff
@@ -38,6 +38,7 @@ <h1>Doki Theme</h1>
     </label>
   </form>
 </div>
+<span id="show-settings">Show Settings</span>
```

Comment:
Wasn't sure what element to use, so I stuck with span and made it look like  link 😅

Reply:
Ehh, it works.

To make it a bit better though, there should be some kind of affordance here to make it clear it's a clickable item. For instance, adding some hover effects or something. 

Honestly, I prefer the Heart interaction over the link addition. In any case, so long as the Heart navigation is still there I'm fine with this 👍

Reply:
> Honestly, I prefer the Heart interaction over the link addition

This is just me, but I didn't know to click it until you said something about it, when I see a link, I click and expect to go places. Perhaps because there maybe wasn't anything, other than the cursor changing to button click mode, that this might take me someplace.